modules/RTC/JitsiRemoteTrack.js

```diff
@@ -51,7 +85,7 @@ JitsiRemoteTrack.prototype.setMute = function (value) {
         this.stream.muted = value;
 
     this.muted = value;
-    this.eventEmitter.emit(JitsiTrackEvents.TRACK_MUTE_CHANGED);
+    this.eventEmitter.emit(JitsiTrackEvents.TRACK_MUTE_CHANGED, this);
```

Comment:
Why do you need this parameter? You have to attach the listener to the track and this means that the listener knows which is the track.